java/fundamentals/intro-to-java/java-hello-world.md

```diff
@@ -1,71 +1,62 @@
 ---
 author: mihaiberq
 
-levels:
-
-  - beginner
-
 type: normal
 
 category: must-know
 
 aspects:
-
   - introduction
 
-inAlgoPool: false
-
-
 links:
-
   - '[Get started with Java](https://docs.oracle.com/javase/tutorial/getStarted/cupojava/index.html){website}'
+  - '[Playground](https://repl.it/@enkicontent/HelloJavaPlayground){website}'
 
 ---
 
-# `Hello World!`
+# Hello Java!
 
 ---
 ## Content
 
-Now that you know a little bit about Java, it is time to show you how to build your first app. 
+Now that you know a little bit about Java, it's time to show you how to build your first app. 
 
 There are a couple of choices when it comes to where to write Java code: simple text editors (Atom, Sublime, Emacs, Vim) or IDEs[1] (IntelliJ, Eclipse). You can download all of them for free.
 
 Probably the most used first code snippet for any language is the `hello world` one. Here it is:
 ```java
-public class HelloWorld {
+// HelloJava.java
+public class HelloJava {
   public static void main(String[] args) {
-    System.out.println("Hello World!");
+    System.out.println("Hello Java!");
   }
 }
-// Hello World!
+// Hello Java!
 ```
 
-All the code does is print `Hello World!` to the console. As it is hard to represent the whole environment on a phone, we will assume that text after `//` is the console output, unless stated otherwise.
+Wow, that's a lot! All this does is display `"Hello Java!"`.
 
-For any Java code to work, a `class` has to enclose it. In our case, the class is called `HelloWorld`. **Keep in mind** that Java requires the file containing the class `HelloWorld` *to be called* `HelloWorld.java`.
+As it is hard to represent the whole environment on a phone, we will assume that text after `//` is the displayed output, unless stated otherwise.
```

Comment:
```suggestion
As it is hard to represent the whole environment on a phone, we will assume that text after `//` is the displayed output after executing the program, unless stated otherwise.
```